dl/src/main/scala/com/intel/analytics/bigdl/models/vgg/Train.scala

```diff
@@ -94,6 +94,10 @@ object Train {
         .setState(state)
         .setEndWhen(Trigger.maxEpoch(param.maxEpoch))
         .optimize()
+
+      if (sc.isDefined) {
```

Comment:
Usually it will, but I met a issue the application hung after failed to save model. I think it's a good idea to explicitly call sc.stop() to avoid application hung, just as spark example code do.